src/views/Onboarding.vue

```diff
@@ -694,7 +694,7 @@ export default {
         navigator.share({
           title: 'Ambianic Edge Device Access Request',
           text:
-            'Hi Blob, please send me an access invitation to your Ambianic Edge Device'
+              'Hi Blob, please send me an access invitation to your Ambianic Edge Device'
```

Comment:
The same request text appears twice in the same file. One is now using ___ and the other Blob....Is there a better way to handle repetitive literals?